src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -94,7 +93,7 @@ void allFalse() throws Exception {
     }
 
     @Test
-    void emptyIterator() throws Exception {
+    void emptyIterable() throws Exception {
         new Assertion<>(
             "Iterator must be empty",
```

Comment:
@victornoel I think Assertion's comment should be `Iterable must be empty` instead of `Iterator must be empty`.